core-web/libs/template-builder/src/lib/components/template-builder/components/add-style-classes-dialog/add-style-classes-dialog.component.ts

```diff
@@ -102,4 +106,16 @@ export class AddStyleClassesDialogComponent implements OnInit, AfterViewInit, On
     onUnselect(): void {
         this.store.removeLastClass();
     }
+
+    /**
+     * @description Used to listen for enter presses
+     *
+     * @param {KeyboardEvent} event
+     * @memberof AddStyleClassesDialogComponent
+     */
+    onKeyUp(event: KeyboardEvent): void {
+        if (event.key === 'Enter' && this.autoCompleteInput.value) {
+            this.autoComplete.selectItem({ cssClass: this.autoCompleteInput.value });
```

Comment:
Using this method makes sure that:

1. The option is selected
2. Suggestions are hidden
3. The text input is cleared